src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IArgument.cs

```diff
@@ -2214,6 +2221,34 @@ struct G<T>
             VerifyOperationTreeAndDiagnosticsForTest<InvocationExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void GettingInOutConverionFromCSharpArgumentShouldThrowException()
+        {
+            string source = @"
+class P
+{
+    static void M1()
+    {
+        /*<bind>*/M2(1)/*</bind>*/;
+    }
+
+    static void M2(int x)
+    {
+    }
+}
+";
+            var compilation = CreateCompilation(source);
+            var (operation, syntaxNode) = GetOperationAndSyntaxForTest<InvocationExpressionSyntax>(compilation);
+
+            var invocation = (IInvocationExpression)operation;
+            var argument = invocation.ArgumentsInEvaluationOrder[0];
+
+            Assert.Throws<ArgumentException>(() => argument.GetInConversion());
+            Assert.Throws<ArgumentException>(() => argument.GetOutConversion());
+        }
```

Comment:
Where are `GetInConversion` and `GetOutConversion` defined? And why do they throw `ArgumentException` in this case?

Reply:
Those are extension methods for VB only (defined in MS.CA.VB.dll), as it returns VB `Conversion`. In this test we pass in an IArgument from C# code, therefore it throws.

Reply:
Are these `VisualBasicExtension` methods? If so, perhaps add a comment here so it's clear why we expect exceptions to be thrown.